moped-database/metadata/tables.yaml

```diff
@@ -4072,68 +4078,8 @@
           - public_process_status_id
         filter: {}
         check: {}
-  event_triggers:
-    - name: activity_log_moped_project
-      definition:
-        enable_manual: false
-        insert:
-          columns: '*'
-        update:
-          columns:
-            - ecapris_subproject_id
-            - project_id
-            - date_added
-            - added_by
-            - is_deleted
-            - project_sponsor
-            - parent_project_id
-            - interim_project_id
-            - project_lead_id
-            - project_name
-            - project_description
-            - project_website
-            - knack_project_id
-            - public_process_status_id
-      retry_conf:
-        interval_sec: 10
-        num_retries: 0
-        timeout_sec: 60
-      webhook_from_env: HASURA_ENDPOINT
-      headers:
-        - name: x-hasura-admin-secret
-          value_from_env: ACTIVITY_LOG_API_SECRET
-      request_transform:
-        body:
-          action: transform
-          template: |-
-            {
-                            "query": "mutation InsertActivity($object: moped_activity_log_insert_input!, $project_id:Int!, $updated_at:timestamptz ) { insert_moped_activity_log_one(object: $object) { activity_id } update_moped_project_by_pk(pk_columns: {project_id: $project_id}, _set: {updated_at: $updated_at}) { updated_at }}",
```

Comment:
My note in the issue description about removing this event was misleading, because we do want to remove the `update_moped_project_by_pk` mutation, but we need to preserve the `insert_moped_activity_log_one` mutation that inserts into the activity log.

Reply:
Ah, got it! Thank you for the clarification John. I'll get this fixed up right after dev sync. 🙏

Reply:
Got it, I think! I went back and forth more times than I want to admit because it was throwing errors if i left the `project_id` and `updated_at` values in the variables payload and also broken if i took them out. Finally,  I realized that I had forgotten to take them out of the graphql arguments themselves. 🤦

GraphQL still gives me a turn-around sometimes. A query language that comes in the form of functions. It's hard for old dogs to learn new tricks!